src/librustc/diagnostics.rs

```diff
@@ -1943,6 +1943,20 @@ fn main() {
 ```
 "##,
 
+E0452: r##"
+An invalid lint attribute has been given. Erroneous code example:
+
+```
+#![allow(foo = "")] // error: malformed lint attribute
+```
+
+You need to pass an ident as lint attribute to fix this issue:
```

Comment:
"Lint attributes only accept a list of identifiers (where each identifier is a lint name). Ensure the attribute is of this form:"